pages/scholars/_id.vue

```diff
@@ -1,13 +1,300 @@
+                      <v-img
+                          src="https://assets.coingecko.com/coins/images/10366/large/SLP.png?1578640057"
+                          max-height="35"
+                          max-width="35"
+                      ></v-img>
+                    <div class="headline-1 mx-3">
+                      {{ farmedToday }} 
+                    </div>
+                  </v-row>
+                </v-card-title>
+                <v-row justify="center">
+                  <v-card-title>Farmed Today</v-card-title>
+                </v-row>
+            </v-card>
+        </v-col>
+        <v-col
+            cols="12"
+            md="2"
+            sm="2"
+        >
+            <v-card
+              min-height="70"
+            >
+                <v-card-title>
+                  <v-row
+                    class="pt-5 px-5"
+                    align="center"
+                    justify="center"
+                  >
+                    <v-img
+                      class="rounded-circle"
+                      src="https://chimeratribune.com/wp-content/uploads/2020/11/axieareanaswords.png"
+                      max-height="35"
+                      max-width="35"
+                    ></v-img>
+                    <div class="headline-1 mx-3">
+                      {{ mmr }}
+                    </div>
+                  </v-row>
+                </v-card-title>
+                <v-row justify="center">
+                  <v-card-title>Current MMR</v-card-title>
+                </v-row>
+            </v-card>
+        </v-col>
+      </v-row>
+      <v-divider class="mt-10"></v-divider>
+      <div class="mt-5">
+        <h2>
+        {{axies.length}} Axies
+        </h2>
+      </div>
+      <v-row
+        align="center"
+        justify="center"
+      >
+        <v-col
+          cols="12"
+          md="2"
+          v-for="axie in axies"
+          :key="axie.id"
+        >
+          <v-card>
+            <v-card-subtitle>
+              Axie #{{ axie.id }}
+            </v-card-subtitle>
+            <v-card-text
+              align="center"
+              justify="center"
+            >
+              <v-img
+                :src="axie.image"
+                max-height="250"
+                max-width="250"
+              >
+              </v-img>
+            </v-card-text>
+            
+          </v-card>
+        </v-col>
+      </v-row>
   </app-main-container>
 </template>
 
 <script>
-export default {};
+  export default{
+    data() {
+      return {
+        slp: "--",
+        dailySLP: "--",
+        farmedToday: "--",
+        lastClaimedSLP: 0,
+        mmr: "--",
+        axies: []
+      }
+    },
+    async created() {
+      await this.getUserAxies()
+      await this.getUserInfo()
+      this.getDailySLP()
+    },
+    methods: {
+      async getUserAxies() {
+        await this.$axios.$get('https://axie-infinity.p.rapidapi.com/get-axies/0xc20dabcad7bf3971fb11e89bf50bf2ff6dec0fd3', {
```

Comment:
The base API URL should not be hard-coded in here and should be moved into a single source of truth where we'll import it and use it wherever we require to make calls for this API.

In the root directory of the project, create a directory and call it `constants` and inside this directory create a new file called `index.js` (`./constants/index.js`). And inside the file have the following exports
```js
export const AXIE_RAPID_API_BASE_URL = 'https://axie-infinity.p.rapidapi.com'
```

Then from the current file for User detail page/view import it in

```js
<script>
import { AXIE_RAPID_API_BASE_URL } from '~/constants/index.js'
// if it doesn't work then use relative path instead '../../constants/index.js'

export default {
  methods: {
   getAxie(roninWalletAddress) {
      this.$axios.$get(`${AXIE_RAPID_API_BASE_URL }/get-axies/${roninWalletAddress}`)
   }
  }
}
</script>
```